cli/src/cluster_query.rs

```diff
@@ -1342,6 +1359,32 @@ pub fn process_get_transaction_count(rpc_client: &RpcClient, _config: &CliConfig
     Ok(transaction_count.to_string())
 }
 
+// if `--additional-fee` is not specified, no comput-budget instruction will be included in
+// transaction;
+// otherwise, user specified additional-fee lamports will be used for compute-budget instruction;
+// if `0` is specified, the max-fee of one signature will be used as default.
+fn build_compute_budget_instruction(additional_fee: &Option<u32>) -> Option<Instruction> {
+    let mut additional_fee = (*additional_fee)?;
+
+    // request default compute units, defined in program-runtime/src/compute_budget.rs
+    let request_units = 200_000u32;
+
+    if additional_fee == 0u32 {
+        let signature_count = 1;
+        let write_lock_count = 1;
+        let default_fee_structure = FeeStructure::default();
+        additional_fee = default_fee_structure
+            .get_max_fee(signature_count, write_lock_count)
+            .try_into()
+            .unwrap();
```

Comment:
Can we make this an `expect()` and explain why we think `get_max_fee()` won't ever return > u32::MAX?

Reply:
yep, remove this block of code